packages/components-js/tests/e2e/puppeteer/specs/pin-code.e2e.ts

```diff
@@ -87,6 +88,15 @@ describe('label', () => {
   });
 });
 
+describe('should render correct amount of inputs', () => {
+  it.each(PIN_CODE_LENGTHS)('length=%d', async (length) => {
```

Comment:
Should probably switch this a bit around wording wise to match the other tests.
<img width="1043" alt="Bildschirmfoto 2023-10-30 um 12 10 58" src="https://github.com/porsche-design-system/porsche-design-system/assets/4380295/e43cc945-ae04-4257-a80e-2887f1109ff9">